compiler/resolution/functionResolution.cpp

```diff
@@ -6914,9 +7226,7 @@ requiresImplicitDestroy(CallExpr* call) {
 
         // Return type is a record (which includes array, record, and
         // dist) or a ref counted type that is passed by reference
```

Comment:
I don't see anything wrong with that comment but maybe I am missing something you see?